cloud-chatops/version.txt

```diff
@@ -1 +1 @@
-2.0.1
```

Comment:
One thing to watch out for is that the version here is the same as in this PR: https://github.com/stfc/cloud-docker-images/pull/50

Reply:
I have left them both as the same version. When one of them (either) is merged then the other will be rebased and updated. If I were to change this PR to 2 versions ahead (2.0.3) there would be a chance of a version being skipped.